### PR TITLE
Ocm add volsync addon controller

### PR DESCRIPTION
**Pull request by tesshuflower**

### CODE REVIEW

**Review by schmidtd — CHANGES_REQUESTED** [review]

File: ci-operator/config/stolostron/volsync-addon-controller/stolostron-volsync-addon-controller-release-2.5.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      IMAGE_REPO: volsync-addon-controller
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_IMAGE_REMOTE_REPO_SRC="registry.ci.openshift.org/stolostron"
+        export SELF="make -f Makefile.prow"
+        export OSCI_PUBLISH_DELAY="0"
+        export OSCI_COMPONENT_NAME="volsync-addon-controller"
+        make -f Makefile.prow osci/publish
```
[the remark this answers is not in the document]
Comment (tesshuflower) [reply ci-operator/config/stolostron/volsync-addon-controller/stolostron-volsync-addon-controller-release-2.5.yaml:72]: Done!

**Review by schmidtd — APPROVED** [review]
/lgtm

**Pull request merged**